server/views/components/place-context-header/macro.njk

```diff
@@ -1,38 +1,59 @@
+                        </div>
+                        {% if person.genderIdentity %}
+                            <div class="place-context-header__row">
+                                <dt class="place-context-header__key">Gender identity: </dt>
+                                <dd class="place-context-header__value">{{ person.genderIdentity }}</dd>
+                            </div>
+                        {% else %}
+                            <div class="place-context-header__row">
+                                <dt class="place-context-header__key">Sex: </dt>
+                                <dd class="place-context-header__value">{{ person.sex }}</dd>
+                            </div>
+                        {% endif %}
+                    {% endif %}
+
+                    {% if placeContext.assessment.accommodationRequiredFromDate %}
+                        <div class="place-context-header__row">
+                            <dt class="place-context-header__key">Accommodation required: </dt>
+                            <dd class="place-context-header__value">
+                                {% set accommodationRequiredFromDate = formatDate(placeContext.assessment.accommodationRequiredFromDate, {format: 'short'}) %}
+                                {{ accommodationRequiredFromDate }}
+                            </dd>
+                        </div>
                     {% endif %}
-                {% endif %}
-                <div>CRN: {{ person.crn }}</div>
-                {% if placeContext.assessment.accommodationRequiredFromDate %}
-                    {% set accommodationRequiredFromDate = formatDate(placeContext.assessment.accommodationRequiredFromDate, {format: 'short'}) %}
-                    <div>Accommodation required from: {{ accommodationRequiredFromDate }}</div>
-                {% endif %}
-                {% if placeContext.assessment.summaryData.isAbleToShare === true %}
-                    <div>Suitable to share: Yes</div>
-                {% endif %}
-                {% if placeContext.assessment.summaryData.isAbleToShare === false %}
-                    <div>Suitable to share: No</div>
-                {% endif %}
+
+                    <div class="place-context-header__row">
+                        <dt class="place-context-header__key">Suitable to share: </dt>
+                        <dd class="place-context-header__value"> 
+                            {{ "Yes" if placeContext.assessment.summaryData.isAbleToShare === true else "No" }}
+                        </dd>
+                    </div>
+                </dl>
+
+                <a class="govuk-link govuk-link--inverse" href="{{ addPlaceContext(paths.assessments.summary({ id: placeContext.assessment.id })) }}" rel="noreferrer noopener" target="_blank">
+                View referral summary (opens in new tab)
+                </a>
```

Comment:
Although not in prototype, this link probably _is_ useful -- to be confirmed with designer

Reply:
If only we had analytics to see if users are actually clicking this link now.

I only left that in there because it was present in the existing component so there must of been a need for it in the first place.

Reply:
`Suitable for sharing` was also not included in the prototype but I left it in as I thought was needed.

Reply:
Yes, I was agreeing with keeping content as is -- so leaving this link and `Suitable for sharing` seems the right thing to do at this point.